sdk/core/azure-core/test/ut/transport_adapter_implementation.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "transport_adapter_base.hpp"
+#include <azure/core/context.hpp>
+#include <azure/core/http/policy.hpp>
+#include <azure/core/response.hpp>
+
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include <http/curl/curl_connection_pool_private.hpp>
+#include <http/curl/curl_connection_private.hpp>
```

Comment:
```suggestion
#include <azure/core/http/curl/curl_connection_pool_private.hpp>
#include <azure/core/http/curl/curl_connection_private.hpp>
```

Reply:
No. These are private headers. The root folder is in `sdk/core/azure-core/src`.
I will add a comment about it